next-tavla/app/(admin)/edit/[id]/components/Footer/index.tsx

```diff
@@ -0,0 +1,40 @@
+'use client'
+import { TextField } from '@entur/form'
+import { ValidationInfoIcon } from '@entur/icons'
+import { Heading3 } from '@entur/typography'
+import { Tooltip } from 'app/(admin)/components/Tooltip'
+import { SubmitButton } from 'components/Form/SubmitButton'
+
+function Footer({
+    footer,
+    action,
+}: {
+    footer?: string
+    action: (data: FormData) => void
+}) {
+    return (
+        <form className="box flexColumn justifyBetween g-2" action={action}>
```

Comment:
skal vi legge en toast et sted? da blir det likt med de andre meta innstillingene på edit sidene

Reply:
Ja, jeg tenkte tanken. Kunne eventuelt hatt en switch (som med gåavstand) for å velge om den skal vises eller ikke.